packages/components/src/typography/src/Heading.css

```diff
@@ -1,44 +1,64 @@
 :where(.o-ui-heading) {
     margin: 0;
     font-size: inherit;
-    font-weight: 400;
-    font-variation-settings: "wght" 600;
+    font-weight: 680;
+    font-family: var(--hop-font-family-primary);
```

Comment:
Why did you opt for core tokens here?

Reply:
We currently don't have a generic semantic tokens for headings. https://hopper.workleap.design/tokens/semantic/font-family it felt weird to use any of these as they were tied to a size. This declaration is generic.